sky/clouds/do.py

```diff
@@ -0,0 +1,275 @@
+""" Digital Ocean Cloud. """
+
+import json
+import typing
+from typing import Dict, Iterator, List, Optional, Tuple
+
+from sky import clouds
+from sky.clouds import service_catalog
+from sky.provision.do import utils
+from sky.utils import resources_utils
+
+if typing.TYPE_CHECKING:
+    from sky import resources as resources_lib
+
+_CREDENTIAL_FILE = 'config.yaml'
+
+
+@clouds.CLOUD_REGISTRY.register
+class DO(clouds.Cloud):
+    """Digital Ocean Cloud"""
+
+    _REPR = 'DO'
+    _CLOUD_UNSUPPORTED_FEATURES = {
```

Comment:
Do we supprot opening ports on DO? Seems like we don't have corresponding implementation in provisioner.

Reply:
ports are exposed by default on digital ocean instances so you don't need any further action to open ports. resolving

Reply:
It will be good to add a comment here ;)

https://github.com/skypilot-org/skypilot/blob/800f7d6971bd604f266faebb33d044c7d5baca55/sky/provision/fluidstack/instance.py#L331-L332